tools/csv-gen/csv-gen.go

```diff
@@ -150,7 +150,7 @@ func main() {
 		opVersion.Version = semver.MustParse(version.Version)
 		templateStruct.Spec.Version = opVersion
 		templateStruct.Spec.Replaces = operatorName + "." + version.PriorVersion
-		templateStruct.Spec.Description = descrip
+		templateStruct.Spec.Description = "Deploys and manages Red Hat Process Automation Manager and Red Hat Decision Manager environments.\n\n* **Red Hat Process Automation Manager** is a platform for developing containerized microservices and applications that automate business decisions and processes. It includes business process management (BPM), business rules management (BRM), and business resource optimization and complex event processing (CEP) technologies. It also includes a user experience platform to create engaging user interfaces for process and decision services with minimal coding.\n\n * **Red Hat Decision Manager** is a platform for developing containerized microservices and applications that automate business decisions. It includes business rules management, complex event processing, and resource optimization technologies. Organizations can incorporate sophisticated decision logic into line-of-business applications and quickly update underlying business rules as market conditions change.\n\n[See more](https://www.redhat.com/en/products/process-automation)."
```

Comment:
maybe change to something like this so reusing `descrip` var?
```golang
templateStruct.Spec.Description = descrip + "\n\n* **Red Hat Process Automation Manager** is a platform for developing containerized microservices and applications that automate business decisions and processes. It includes business process management (BPM), business rules management (BRM), and business resource optimization and complex event processing (CEP) technologies. It also includes a user experience platform to create engaging user interfaces for process and decision services with minimal coding.\n\n * **Red Hat Decision Manager** is a platform for developing containerized microservices and applications that automate business decisions. It includes business rules management, complex event processing, and resource optimization technologies. Organizations can incorporate sophisticated decision logic into line-of-business applications and quickly update underlying business rules as market conditions change.\n\n[See more](https://www.redhat.com/en/products/process-automation)."
```

Reply:
did `make csv` produce the same results?

Reply:
Yes, i did change createdAt: "2020-02-26 09:14:55" which I ignored.